test/uk/gov/hmrc/apiplatformoutboundsoap/services/OutboundServiceSpec.scala

```diff
@@ -316,6 +322,40 @@ class OutboundServiceSpec extends AnyWordSpec with Matchers with GuiceOneAppPerS
       verify(outboundMessageRepositoryMock).updateStatus(retryingMessage.globalId, SendingStatus.FAILED)
       verify(outboundMessageRepositoryMock, never).updateNextRetryTime(*, *)
     }
+
+    "notify the caller on the notification URL supplied that the retrying message is now SENT" in new Setup {
+      val retryingMessage = RetryingOutboundSoapMessage(randomUUID, Some("MessageId-A1"), "<IE4N03>payload</IE4N03>", DateTime.now(UTC), DateTime.now(UTC))
+      val sentMessageForNotification: SentOutboundSoapMessage = retryingMessage.toSent
+      when(appConfigMock.parallelism).thenReturn(2)
+      when(appConfigMock.parallelism).thenReturn(2)
+      when(appConfigMock.retryInterval).thenReturn(Duration("1s"))
+      when(outboundConnectorMock.postMessage(*)).thenReturn(successful(OK))
+      when(outboundMessageRepositoryMock.updateStatus(*, *)).thenReturn(successful(Some(sentMessageForNotification)))
+      when(outboundMessageRepositoryMock.retrieveMessagesForRetry).
+        thenReturn(fromFutureSource(successful(fromIterator(() => Seq(retryingMessage).toIterator))))
+
+      await(underTest.retryMessages)
+
+      verify(notificationCallbackConnector).sendNotification(refEq(sentMessageForNotification))(*)
```

Comment:
We could add a test where this call fails but the retrying message is still set to SENT

Reply:
Tested in a new scenario added to this spec class. Also renamed one mock to make its mockness clearer